content/applications/inventory_and_mrp/inventory/management/misc/owned_stock.rst

```diff
@@ -1,45 +1,106 @@
 
-When in the *Inventory* app, open the receipts and create a new
-reception. On the right side, you will see that a new line called
-*Assign Owner* has appeared. There, you can specify the partner which
-owns the stock.
+Sell and Deliver Consignment Stock
+==================================
 
-.. image:: owned_stock/owned_stock_02.png
-    :align: center
+Once it's been received into the warheouse, consignment stock can be sold the same as any other
+product. To create a sales order, navigate to the :guilabel:`Sales` app, and from the
+**Quotations** overview, click :guilabel:`New`.
+
+Fill out all the information on the new quotation, and choose a customer to enter into the
+:guilabel:`Customer` field.
 
 .. note::
-   If you are the owner, you can leave the field blank.
+    The :guilabel:`Customer` *must* be different from the vendor that supplied the consignment
+    stock received into and stored in the warheouse.
+
+Add the consignment product to the product lines, set the :guilabel:`Quantity`, and once everything
+has been filled out, :guilabel:`Confirm`.
+
+.. image:: owned_stock/owned-stock-sales-order.png
+   :align: center
+   :alt: Sales order of consignment stock.
+
+Once the quotation has been confirmed, it will become a sales order. From here, the products can be
+delivered and invoiced for.
+
+Traceability and Reporting of Consignment Stock
+===============================================
+
+Although consignment stock is owned by the vendor who supplied it and not by the company storing
+it in their warehouse, consignment products will still appear in certain inventory reports. To find
+inventory reports, go to :menuselection:`Inventory --> Reporting`, and choose a report to view.
+
+.. note::
+    Since the consignee does not actually own consigment stock, these products are *not* reflected
+    in the **Stock Valuation** report, and have no impact on the consignee's inventory valuation.
+
+.. image:: owned_stock/owned-stock-moves-history.png
+   :align: center
+   :alt: Consignment stock moves history.
+
+**Stock** Report
+----------------
+
+To view the **Stock** report, go to :menuselection:`Inventory --> Reporting --> Stock`. From this
+report, information about all stock on-hand can be seen. For consignment products, the information
+in this report is the same as any other product: the history of its product moves can be reviewed;
+any replenishment rules are listed; its **Locations** are available; and its :guilabel:`Forecasted`
+units are listed, as well.
 
-Once the receipt is validated, the products enter your stock but still
-belong to the owner. They don’t impact your inventory valuation.
+**Locations** Report
+--------------------
 
-.. image:: owned_stock/owned_stock_03.png
-    :align: center
+To view the **Locations** report, go to :menuselection:`Inventory --> Reporting --> Locations`.
+From this report, the **Locations** of all stock on-hand are displayed, as well as the quantities
```

Comment:
From this report, the :guilabel:`Locations` of all stock on-hand are displayed, in addition to the quantities per location.